website/static/js/nodesPrivacy.js

```diff
@@ -0,0 +1,272 @@
+/**
+ * Controller for changing privacy settings for a node and its children.
+ */
+'use strict';
+
+var $ = require('jquery');
+var ko = require('knockout');
+var bootbox = require('bootbox');
+var Raven = require('raven-js');
+
+var oop = require('./oop');
+var $osf = require('./osfHelpers');
+var Paginator = require('./paginator');
+var osfHelpers = require('js/osfHelpers');
+var $ = require('jquery');
+var m = require('mithril');
+var Treebeard = require('treebeard');
+var $osf = require('js/osfHelpers');
+var NodesPrivacyTreebeard = require('js/nodesPrivacySettingsTreebeard');
+
+var NODE_OFFSET = 25;
```

Comment:
It looks like this isn't being used anymore -- if that's the case can you remove it?